c2corg_api/views/user.py

```diff
@@ -116,20 +120,128 @@ def post(self):
             log.warning('Error persisting user', exc_info=True)
             raise HTTPInternalServerError('Error persisting user')
 
-        # also create a version for the profile
-        DocumentRest.create_new_version(user.profile, user.id)
```

Comment:
Good question. Was willing to discuss it with you.
Wouldn't it make sense to delay the creation of the profile to the moment when the user is validated? I guess it is not really what I did; how could it be done?